cumulusci/tasks/release_notes/task.py

```diff
@@ -108,96 +109,77 @@ class ParentPullRequestNotes(BaseGithubTask):
 
-    def _handle_parent_branch_name_option(self, generator, parent_branch_name):
+    def _has_parent_branch(self):
+        # TODO: feature_prefix = self.project_config...
+        return self.branch_name.startswith("feature/") and "__" not in self.branch_name
 
-        pull_requests = get_pull_requests_with_base_branch(
-            self.repo, self.repo.default_branch, parent_branch_name
-        )
+    def _commit_is_merge(self):
+        return len(self.commit.parents) > 1
 
-        if len(pull_requests) == 0:
-            self.logger.info(
-                "No pull request found for branch: {}. Exiting...".format(
-                    parent_branch_name
-                )
-            )
-            return
-        elif len(pull_requests) > 1:
-            self.logger.info(
-                "More than one pull request returned with base='master' for branch {}".format(
-                    parent_branch_name
-                )
-            )
-            return
-        else:
-            # We can ONLY aggregate child change notes when given the parent_branch option
-            #
-            # We aren't able to append to the 'Unaggregated Pull Requests' header.
-            # We don't know at what time the label was applied to the pull request;
-            # and therefore, we cannot determine which child pull requests are already
-            # aggregated into the parent pull request, and which ones should be
-            # included in the 'Unaggregated Pull Requests' section.
-            generator.aggregate_child_change_notes(pull_requests[0])
-
-    def _get_parent_pull_request(self, branch_name):
+    def _get_parent_pull_request(self):
         """Attempts to retrieve a pull request for the given branch.
         If one is not found, then it is created and the 'Build Change Notes' 
         label is applied to it."""
         requests = get_pull_requests_with_base_branch(
-            self.repo, self.repo.default_branch, branch_name
+            self.repo, self.repo.default_branch, self.branch_name
         )
         if len(requests) == 0:
             self.logger.info(
                 "Pull request not found. Creating pull request for branch: {} with base of 'master'.".format(
-                    branch_name
+                    self.branch_name
                 )
             )
-            parent_pull_request = create_pull_request(self.repo, branch_name)
+            parent_pull_request = create_pull_request(self.repo, self.branch_name)
             add_labels_to_pull_request(
-                self.repo, parent_pull_request, self.build_notes_label
+                self.repo, parent_pull_request, self.options.get("build_notes_label")
             )
         else:
             parent_pull_request = requests[0]
         return parent_pull_request
+
+    def _get_child_branch_name_from_merge_commit(self):
+        pull_requests = get_pull_requests_by_commit(
+            self.github, self.repo, self.project_config.repo_commit
```

Comment:
Does this work? I expected we would need to search for one of the parent commits, not the merge commit, since the child branch doesn't include the merge commit.